.rubocop.yml

```diff
@@ -1,7 +1,7 @@
 # Defaults can be found here: https://github.com/bbatsov/rubocop/blob/master/config/default.yml
 
 AllCops:
-  TargetRubyVersion: 2.0
+  TargetRubyVersion: 2.3
```

Comment:
Seems legit

Reply:
I needed it to stop rubocop complaining, and the gemspec says 2.3 is the minimum supported version anyway 😉